examples/object-oriented-cpp/PineAPPL.hpp

```diff
@@ -226,7 +251,7 @@ struct Grid {
      */
     std::string get_key_value(const std::string &key) const {
         auto* value = pineappl_grid_key_value(this->raw, key.c_str());
-        const std::string res(value);
+        std::string res(value);
```

Comment:
According to `clang-tidy` the modifier `const` prevents a returned value optimization.